blas/gonum/single_precision.bash

```diff
@@ -35,6 +35,7 @@ cat level1float64.go \
       -e 's_^// Id_// Is_' \
       -e 's_"gonum.org/v1/gonum/internal/asm/f64"_"gonum.org/v1/gonum/internal/asm/f32"_' \
       -e 's_"math"_math "gonum.org/v1/gonum/internal/math32"_' \
+      -e 's_safmin = 0x1p-1022_safmin = 0x1p-126_' \
```

Comment:
I'm not completely happy with this, it's too specific but I can't quickly think of a better solution. I thought about having a file like `constants.go` with `dsafmin, dsafmax, ssafmin, ssafmax` and then here change from the d-prefix to the s-prefix. However, having these constants even only at package level feels like making them too public, they won't be needed elsewhere in BLAS.

Reply:
We can live with it for the time being. I'm still wondering how we will deal with precision-dependent constants when we shift to a generics-based approach (assuming we do, but it's something that I'd like to do since it will reduce line count and improve testing).

Reply:
Ok, I'll leave it in. Regarding generics, I'm not following that topic much, not enough bandwidth, but sounds like something that would be nice to do.